hr_expense_petty_cash/views/petty_cash_views.xml

```diff
@@ -65,7 +65,30 @@
         <field name="model">petty.cash</field>
         <field name="arch" type="xml">
             <search string="Petty Cash">
-                <group expand="0" string="Group By" />
+                <field
```

Comment:
Please delete it, Standard with petty cash version13 is not group by.
I think we should add new search group by on PR https://github.com/OCA/hr/pull/924 and help improve it for version 13 and 14